packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -373,9 +374,17 @@ export class StatsAnalyzer extends EventsScope {
         name: MEDIA_DEVICES.MICROPHONE,
       });
     }
-    if (this.statsResults['video-send']) {
+
+    let existingVideoSender;
+    Object.keys(this.statsResults).forEach((item) => {
+      if (item.includes('video-send')) {
+        existingVideoSender = this.statsResults[item];
```

Comment:
Should we break after this? From what I can tell, there can be multiple stats items with `video-send`, and `existingVideoSender` would be overwritten every time.

Alternatively, if we only care about getting the first item, we could use `Array.find()`.

Reply:
Updated.